### PR TITLE
IDE-3340 Liferay IDE check server status error for liferay jboss portal 

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

**Comment by yanan** [comment]
Build Failed.
 220 tests run, 14 skipped, 1 failed.
Test Failed on Jenkins http://ide-build-1:9090/ on this commit d1da9fcc527e47d75aa98024184c8538cb3547ea on build 53

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/53/checker.log for detailed testing results

**Comment by yanan** [comment]
Test Failed on Jenkins http://ide-build-1:9090/ on this commit d1da9fcc527e47d75aa98024184c8538cb3547ea on build 53

Refer to https://testing-liferayide.wedeploy.io/ui-testing-reports/53/checker.log for detailed testing results

File: tests-resources/bundles.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
 liferay-ce-portal-tomcat-7.0-ga4-20170613175008905.zip,liferay-ce-portal-7.0-ga4,tomcat-8.0.32,tomcat,7.0-ga4
 liferay-portal-tomcat-6.2-ce-ga6-20160112152609836.zip,liferay-portal-6.2-ce-ga6,tomcat-7.0.62,tomcat,6.2-ga6
+liferay-ce-portal-wildfly-7.0-ga5-20171018150113838.zip,liferay-ce-portal-7.0-ga5,wildfly-10.0.0,wildfly,7.0-ga5/
```
Comment (jtydhr88) [inline tests-resources/bundles.csv]: shouldn't end with /

File: tests-resources/.gitignore [hunk 2]
```diff
@@ -5,4 +5,5 @@ liferay-ce-portal-*.zip
 liferay-dxp-digital-enterprise-*.zip
 liferay-portal-tomcat-6.2-ce-*.zip
 liferay-plugins-sdk-6.2-ce-*.zip
-ivy-cache-7.0.zip
\ No newline at end of file
+ivy-cache-7.0.zip
+liferay-ce-portal-wildfly-7.0-ga5-20171018150113838.zip
```
Comment (jtydhr88) [inline tests-resources/.gitignore]: use **liferay-ce-portal-wildfly-7.0-*.zip** instead

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 3]
```diff
@@ -83,15 +83,12 @@ public IPath getLiferayBundlesPath() {
 						int index = filePath.indexOf("/ui-tests/");
 
 						_liferayBundlesPath = new Path(filePath.substring(0, index) + "/tests-resources");
-					}
-					else {
+					} else {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 4]
```diff
@@ -83,15 +83,12 @@ public IPath getLiferayBundlesPath() {
...
 						_liferayBundlesPath = new Path(filePath).removeLastSegments(3).append("tests-resources");
 					}
+				} catch (IOException ioe) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 5]
```diff
@@ -83,15 +83,12 @@ public IPath getLiferayBundlesPath() {
...
 				}
-				catch (IOException ioe) {
-				}
-			}
-			else {
+			} else {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 6]
```diff
@@ -232,11 +251,9 @@ public File getValidationFolder() {
 			for (int t = 0; t < columns.length; t++) {
 				if (t == 0) {
 					validationMsgs[i].setInput(columns[t]);
-				}
-				else if (t == 1) {
+				} else if (t == 1) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 7]
```diff
@@ -232,11 +251,9 @@ public File getValidationFolder() {
...
 					validationMsgs[i].setExpect(columns[t]);
-				}
-				else if (t == 2) {
+				} else if (t == 2) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 8]
```diff
@@ -270,8 +287,7 @@ public void killGradleProcess() throws IOException {
 
 				Runtime.getRuntime().exec("taskkill /F /PID " + allGradleProcess[0]);
 			}
-		}
-		catch (Exception e) {
+		} catch (Exception e) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 9]
```diff
@@ -306,8 +322,7 @@ public void prepareGeoFile() {
 			String content = "filePath=" + getLiferayBundlesPath().toPortableString() + "/GeoLiteCity.dat";
 
 			FileUtils.write(content, dest);
-		}
-		catch (Exception e) {
+		} catch (Exception e) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java [hunk 10]
```diff
@@ -324,8 +339,25 @@ public void preparePortalExtFile() {
 
 		try {
 			FileUtil.copyFile(source, dest);
+		} catch (Exception e) {
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/EnvAction.java]: break sf

**Comment by jtydhr88** [comment]
hey @simonjhy please do not break current code style, thank.

**Pull request closed without merging**